units/en/unit2/bellman-equation.mdx

```diff
@@ -0,0 +1,55 @@
+# The Bellman Equation: simplify our value estimation [[bellman-equation]]
+
+The Bellman equation **simplifies our state value or state-action value calculation.**
+
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit3/bellman.jpg" alt="Bellman equation"/>
+
+With what we learned so far, we know that if we calculate the \\(V(S_t)\\) (value of a state), we need to calculate the return starting at that state and then follow the policy forever after. **(Our policy that we defined in the following example is a Greedy Policy, and for simplification, we don't discount the reward).**
+
+So to calculate \\(V(S_t)\\), we need to calculate the sum of the expected rewards. Hence:
+
+<figure>
+  <img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit3/bellman2.jpg" alt="Bellman equation"/>
```

Comment:
I think we should show how to calculate the value given a state not in the bellman equation section but also in the section in which we introduce value based methods (two-types-value-based-methods at the end maybe). It could also be a separate section, and could even have an exercise for the reader. 

My concern is that this is the first time the learners put the math from the previous section in practice and at the same time you're introducing bellman.